src/utils/formatters.ts

```diff
@@ -78,4 +78,28 @@ const formatPaymentRequestSortExpression = (
   return sortExpression;
 };
 
-export { formatDate, formatAmount, formatPaymentRequestSortExpression };
+const getDateInPast = (daysToGoBack: number, startOfDay: boolean): Date => {
+  const date = new Date();
+
+  date.setDate(date.getDate() - daysToGoBack);
+
+  if (startOfDay) {
+    date.setHours(0, 0, 0, 0);
+  } else {
+    date.setHours(23, 59, 59, 999);
+  }
+
+  return date;
+};
```

Comment:
This function could be replaced for another `date-fns` helper function called `add`.

The way you'd use the function is:
```jsx
add(yourDateVariable, { days: -4 })
```

Where the `-4` value should be replaced by the amount of days.